src/protect-function-app/README.md

```diff
@@ -0,0 +1,130 @@
+| **Name**                       | `python-azure-function-api`                                          | Suggested value for this sample. <br/> You can change the app name at any time. |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**   | Suggested value for this sample.                                                |
+| **Platform type**              | _None_                                                               | No redirect URI required; don't select a platform.                              |
+| **Scopes defined by this API** | **Scope name**: `Greeting.Read`<br/>**Who can consent?**: **Admins and users**<br/>**Admin consent display name**: `Read API Greetings`<br/>**Admin consent description**: `Allows the user to see greetings from the API.`<br/>**User consent display name**: `Read API Greetings`<br/>**User consent description**: `Allows you to see greetings from the API.`<br/>**State**: **Enabled** | Required scope for this sample. |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Enable Function App Authentication
+
+1. Open the existing Function App in the Azure portal.
+1. Click on **Authentication** and then **Add identity provider**.
+1. Use these settings in your identity provider configuration.
+
+   | Identity provider setting       | Value for this sample app                                | Notes                                                                            |
+   |--------------------------------:|:---------------------------------------------------------|:---------------------------------------------------------------------------------|
+   | **Identity provider**           | **Microsoft**                                            | Required value for this sample.                                                  |
+   | **App registration type**       | **Provide the details of an existing app registration**  | Required value for this sample.                                                  |
+   | **Application (client) ID**     | `<client-id>`                                            | Required value for this sample. <br/> 'Application (client) ID' of app registration in Azure portal - this value is a GUID               |
+   | **Client secret (recommended)** | _None_                                                   | Suggested value for this sample. <br/> This sample doesn't require this feature. |
+   | **Issuer URL**                  | `https://sts.windows.net/<Tenant ID>/v2.0`               | Required value for this sample. <br/> Update to include 'Tenant ID' of your Azure AD instance - this value is a GUID                     |
```

Comment:
This looks suspect, I'd expect it to be something like:

`https://login.microsoftonline.com/<tenant-id>/v2.0`

Which jives with the App Service auth doc:

![image](https://user-images.githubusercontent.com/13394541/151084154-18312665-150d-4b44-95c7-41067dbccd87.png)

Reply:
Yeah, they are both valid.  The sts just happens to be what the portal will fill in automatically in most situations on this UI.  But you're right that we should lead with the login.microsoft.com DNS value instead.  I should have picked up on that in the first pass.  Thanks, will update